src/Server.hs

```diff
@@ -39,6 +39,11 @@ import Configuration qualified as Config
 import GHC.IO.Encoding ()
 import Github qualified
 import WebInterface qualified
+import Prometheus (exportMetricsAsText)
+import Data.Text.Lazy.Encoding (decodeUtf8)
+import Github qualified
+import Prometheus (exportMetricsAsText)
+import WebInterface qualified
```

Comment:
Some of these imports are duplicate or overlapping, and should be in a different import block.

(fourmolu or another autoformatter would just take of this for us... I think I'll make an issue to have that in this repo.)

Reply:
Sorry for not having checked CI before hitting the 'ask for review' button :sweat_smile: